doc/source/tune/examples/pbt_visualization/pbt_visualization.ipynb

```diff
@@ -290,15 +290,15 @@
        "            </tr>\n",
        "            <tr>\n",
        "                <td style=\"text-align: left\"><b>Ray version:</b></td>\n",
-       "                <td style=\"text-align: left\"><b> 3.0.0.dev0</b></td>\n",
+       "                <td style=\"text-align: left\"><b> 2.8.0</b></td>\n",
        "            </tr>\n",
        "            \n",
        "        </table>\n",
        "    </div>\n",
        "</div>\n"
       ],
       "text/plain": [
-       "RayContext(dashboard_url='', python_version='3.8.13', ray_version='3.0.0.dev0', ray_commit='{{RAY_COMMIT_SHA}}', address_info={'node_ip_address': '127.0.0.1', 'raylet_ip_address': '127.0.0.1', 'redis_address': None, 'object_store_address': '/tmp/ray/session_2022-09-14_11-43-30_517456_2265/sockets/plasma_store', 'raylet_socket_name': '/tmp/ray/session_2022-09-14_11-43-30_517456_2265/sockets/raylet', 'webui_url': '', 'session_dir': '/tmp/ray/session_2022-09-14_11-43-30_517456_2265', 'metrics_export_port': 63865, 'gcs_address': '127.0.0.1:65035', 'address': '127.0.0.1:65035', 'dashboard_agent_listen_port': 52365, 'node_id': '7655cd51cd62a60359c7b4de1d1589eed86fddc71de12a517caccc42'})"
+       "RayContext(dashboard_url='', python_version='3.8.13', ray_version='2.8.0', ray_commit='{{RAY_COMMIT_SHA}}', address_info={'node_ip_address': '127.0.0.1', 'raylet_ip_address': '127.0.0.1', 'redis_address': None, 'object_store_address': '/tmp/ray/session_2022-09-14_11-43-30_517456_2265/sockets/plasma_store', 'raylet_socket_name': '/tmp/ray/session_2022-09-14_11-43-30_517456_2265/sockets/raylet', 'webui_url': '', 'session_dir': '/tmp/ray/session_2022-09-14_11-43-30_517456_2265', 'metrics_export_port': 63865, 'gcs_address': '127.0.0.1:65035', 'address': '127.0.0.1:65035', 'dashboard_agent_listen_port': 52365, 'node_id': '7655cd51cd62a60359c7b4de1d1589eed86fddc71de12a517caccc42'})"
```

Comment:
why is the version number (and the unfulfilled commit) printed out in a notebook? is this really required?